packages/core/src/components/Header/Navigation/MenuItem/MenuItem.tsx

```diff
@@ -146,6 +146,7 @@ export const HvHeaderMenuItem = (props: HvHeaderMenuItemProps) => {
           target={itemTarget}
           {...itemProps}
           aria-current={isCurrent}
+          aria-label={item.label}
```

Comment:
This label is already "visible" in the children, so this seems redundant 🤔

Looking at either Chrome DevTools or https://testing-playground.com using `getByRole("link", {name})` the label doesn't seem to be duplicated. Maybe this is a bug in Playwright...

Can we add a unit test validating this works as expected? And a small inline comment explaining why this is needed, so that it's not removed for its _apparent_ redundancy?

Reply:
Added the comment to provide more context. 
As for the unit test, I'm not entirely sure what it should test with the adding of the aria attribute. Any recommendation?

Reply:
A `getByRole("link", { name: "menu_name"})` like you mentioned would be more than enough 👌

Reply:
Done 👍